enterprise/server/backends/distributed/distributed.go

```diff
@@ -1106,12 +1106,6 @@ func (c *Cache) GetMulti(ctx context.Context, resources []*rspb.ResourceName) (m
 			peerRequests[peer] = append(peerRequests[peer], perHashResources[0])
 		}
 		if len(peerRequests) == 0 {
-			stillMissing := make([]string, 0)
-			for h := range hashResources {
-				if _, ok := gotMap[h]; !ok {
-					stillMissing = append(stillMissing, h)
-				}
-			}
```

Comment:
deleting this is kind of funky -- I assume you remove it because stillMissing is not used. Is that a bug?